ansible/roles/docker/tasks/debian.yml

```
@@ -9,12 +9,21 @@
     repo: "deb [arch=amd64] https://download.docker.com/linux/{{ ansible_distribution|lower}} {{ ansible_distribution_release|lower }} stable"
     update_cache: true
 
-- name: install docker
+- name: install Docker CE (Ubuntu Bionic only)
   apt:
     name: "{{ item }}"
     state: present
   with_items:
     - "docker-ce={{ docker_debian_version }}"
```

This can remain a single task if updating the with_items to:
```suggestion
    - "docker-ce={{ docker_xenial_version if ansible_distribution_release == 'xenial' else docker_debian_version }}"
```

---

Ah, nice trick. I can make that change.

---

Change implemented via 37329d6.